### ISSUE TITLE
Cannot read property of 'useTabs' of undefined

### ISSUE THREAD

**Issue by maraisr** [prettier/prettier#3414]
*Command ran:*
```sh
prettier --write "{*,apps/*/src/**/*}.+(ts|js|scss|css|json|md)"
```

*`.prettierrc` file:*
```yml
printWidth: 80
bracketSpacing: true
semi: true
singleQuote: true
tabWidth: 4
trailingComma: none
useTabs: true

overrides:
  - files: "package.json"
    options:
       printWidth: 999
       tabWidth: 2
       useTabs: false
```

*Error:*
```sh
[error] apps/car/src/pages/exclude-drivers/index.ts: TypeError: Cannot read property 'useTabs' of undefined
[error]     at makeAlign (/code/app/node_modules/prettier/index.js:7657:17)
[error]     at makeAlign (/code/app/node_modules/prettier/index.js:7658:11)
[error]     at fits (/code/app/node_modules/prettier/index.js:7700:22)
[error]     at printDocToString$2 (/code/app/node_modules/prettier/index.js:7814:33)
[error]     at formatWithCursor (/code/app/node_modules/prettier/index.js:27649:26)
[error]     at Object.formatWithCursor (/code/app/node_modules/prettier/index.js:27907:12)
[error]     at format (/code/app/node_modules/prettier/bin/prettier.js:20364:21)
[error]     at eachFilename (/code/app/node_modules/prettier/bin/prettier.js:20552:16)
[error]     at ignorer.filter.forEach.filePath (/code/app/node_modules/prettier/bin/prettier.js:20513:9)
[error]     at Array.forEach (<anonymous>)
[error]     at eachFilename (/code/app/node_modules/prettier/bin/prettier.js:20512:8)
[error]     at Object.formatFiles (/code/app/node_modules/prettier/bin/prettier.js:20525:3)
[error]     at Object.run (/code/app/node_modules/prettier/bin/prettier.js:20998:12)
[error]     at Object.<anonymous> (/code/app/node_modules/prettier/bin/prettier.js:21009:5)
[error]     at Module._compile (module.js:641:30)
[error]     at Object.Module._extensions..js (module.js:652:10)
[error]     at Module.load (module.js:560:32)
[error]     at tryModuleLoad (module.js:503:12)
[error]     at Function.Module._load (module.js:495:3)
[error]     at Function.Module.runMain (module.js:682:10)
[error]     at startup (bootstrap_node.js:191:16)
[error]     at bootstrap_node.js:613:3
```

*Node:* v9.2.0
*Prettier:* v1.9.0

**Comment by maraisr** [comment]
Sorry, saw that you already resolved this as part of #3413

**Comment by duailibe** [comment]
The fix was released in 1.9.1!
